packages/core/echo/echo-db/src/host/echo-host.ts

```diff
@@ -49,7 +49,7 @@ export class EchoHost extends Resource {
       db: kv.sublevel('automerge'),
       indexMetadataStore: this._indexMetadataStore,
       // TODO(dmaretskyi): Still needed for data migration -- remove before the next release.
-      directory: storage.createDirectory('automerge'),
+      directory: storage?.createDirectory('automerge'),
```

Comment:
when could this be undefined?

Reply:
It is used for migration only
So in tests it can be undefined because there is no data to migrate, also in new version we want to delete this one